### DISCUSSION TITLE
React Wrap Balancer for Satori?

### DISCUSSION THREAD

**Discussion by anaclumos** [vercel/satori#445] [General]
@shuding, Is there a way to make @shuding's beautiful library [React Wrap Balancer](https://react-wrap-balancer.vercel.app/) work for @shuding's beautiful library [vercel/satori](https://github.com/vercel/satori)? 😄

**Comment by shuding** [comment]
Yep: https://twitter.com/shuding_/status/1629526577485586434

It's `text-wrap: balance` in CSS :)

**Reply by anaclumos** [reply]
Giving 

```
Type '{ textWrap: string; ... 7 more ...; }' is not assignable to type 'Properties<string | number, string & {}>'.

Object literal may only specify known properties, and 'textWrap' does not exist in type 'Properties<string | number, string & {}>'.ts(2322)

index.d.ts(1882, 9): The expected type comes from property 'style' which is declared here on type 'DetailedHTMLProps<HTMLAttributes<HTMLDivElement>, HTMLDivElement>'
```

**Reply by shuding** [reply]
Seems like a TS error as it doesn't think `text-wrap` is valid CSS, maybe you have to ignore it.
